### ISSUE TITLE
Dvm doesn't start after upgrading openssl from version 1.1 to 3

### ISSUE THREAD

**Issue by gepbird** [diced/dvm#5]
Starting dvm with openssl 3 installed: `dvm: error while loading shared libraries: libcrypto.so.1.1: cannot open shared object file: No such file or directory`.
A workaround for this is to download the old openssl files manually, and have version 1.1 and 3 installed at the same time. (Only having the old openssl version would make you unable to upgrade basically anything, or upgrade everything except openssl, but that will break your system.)

**Comment by diced** [comment]
It might need to be rebuilt again, if using the pre built ones, I'll do that very soon.

**Comment by lfmnovaes** [comment]
I'm using arch linux, and  worked for me installing the old version like this:
```
sudo pacman -S openssl-1.1
```

**Comment by gepbird** [comment]
@diced Any updates?

**Comment by gepbird** [comment]
@diced hello?

**Comment by gepbird** [comment]
@diced Is this project dead?

**Comment by yochananmarqos** [comment]
It will build with `openssl-1.1` with these variables:

```
  export OPENSSL_DIR=/usr/lib/openssl-1.1
  export X86_64_UNKNOWN_LINUX_GNU_OPENSSL_LIB_DIR=/usr/lib/openssl-1.1
  export X86_64_UNKNOWN_LINUX_GNU_OPENSSL_INCLUDE_DIR=/usr/include/openssl-1.1
```

This is only a temporary workaround until @diced has a chance to update the upstream dependencies.

**Comment by arduano** [comment]
@gepbird My fork fixes it, you can see it in #7. For now you can have a workaround where you use a custom PKGBUILD pointing to my fork rather than this repo.
If the PR isn't merged, I'll continue developing my fork and hopefully adding more features too, e.g. auto updates on discord launch (with a launcher executable).

**Comment by arduano** [comment]
@diced Would you be interested in more maintainers on this project?

**Comment by gepbird** [comment]
@arduano 
It's great to see someone keeping this project alive, thanks for your work! If @diced doesn't respond you should keep maintaining your fork, and distributing binaries for the main distros (debian, fedora, arch...).

I used to use this project personally because when discord updated their client and Arch had an older version.  "It was my lucky day" and couldn't use it. After knowing this can be turned off by creating a file at `~/.config/discord/settings.json` with the contents `{ "SKIP_HOST_UPDATE": true }` plus using NixOS I don't use dvm.

**Comment by diced** [comment]
https://github.com/diced/dvm/pull/7 I will think about adding maintainers some other day but I want to get this PR merged soon.

**Comment by diced** [comment]
should be fixed on 1.2.0
